doc/src/manual/methods.md

```diff
@@ -679,8 +679,8 @@ When `AbstractArray` subtypes implement the [`Base.IndexStyle`](@ref) trait,
 other functions such as `map` can dispatch on this information to pick
 the best algorithm (see [Abstract Array Interface](@ref man-interface-array)).
 This means that each subtype does not need to implement a custom version of `map`,
-since the generic definitions + trait classes will enable the system to select the fastest version.
```

Comment:
I'm pretty sure this change is incorrect. It is referring to plural definitions, not possessive.

Reply:
Oh...yes. You are absolutely correct. It leaves just the other one which is minor. I will update.

Reply:
> I'm pretty sure this change is incorrect. It is referring to plural definitions, not possessive.

Reply:
Removed the possessive. The only correction left in this PR is the addition of a missing "is". If this is too minor feel free to reject and close this PR. In that case, I will include (ahem..sneak) this minor correction in a  more substantial PR in the future, if any.

Reply:
Minor PRs are better than putting them in unrelated changes. If you have a bunch of doc changes, it can be good to lump them together, but if you put a small doc change in a bigger PR, that tends to just get it stuck waiting for review of a big change. As long as you aren't spamming us with tiny PRs (ie multiple every day), this is great. These PRs take approximately 5 seconds to review, and are completely uncontroversial, so it's really easy to merge them.